.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,60 @@
+name: Bug Report
+description: Create a report to help us improve the game
+labels: ["Type: Bug"]
+body:
+- type: textarea
+  attributes:
+    label: Describe the bug
+    description: A clear and concise description of what the bug is. If you have a video of it happening, you may post it here.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: How To Reproduce
+    description: Steps to reproduce the behavior. If you have a video with the steps to recreate the bug, please post it here.
+    placeholder: |
+      1. Open ...
+      2. Do ...
+      3. See bug
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Expected Behavior
+    description: A clear and concise description of what is expected to happen.
+    placeholder: |
+      When I do X then Z should happen.
+  validations:
+    required: true
+    
+- type: textarea
+  attributes:
+    label: Screenshots
+    description: If applicable, add screenshots to help explain your problem.
+  validations:
+    required: false
```

Comment:
a thought ive had a few times is if we should remove the "screenshot" section completely. at least for how i write issues i'll tend to write words, add an image, then write more words. leaving it blank will make something like this 

https://github.com/braem/issue-testing/issues/10

which is probably fine, just a bit ugly.